runtime/src/cost_model.rs

```diff
@@ -163,16 +169,16 @@ impl CostModel {
         }
 
         // calculate bpf cost based on compute budget instructions
-        let mut budget = ComputeBudget::default();
+        let mut compute_budget = ComputeBudget::default();
 
-        // Starting from v1.15, cost model uses compute_budget.set_compute_unit_limit to
+        // Starting from v1.15, cost model uses compute_compute_budget.set_compute_unit_limit to
```

Comment:
nit: compute_compute_budget

Reply:
Ooops, search/replace beyond scope :) Thanks for pointing out, fixing it